msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -266,19 +266,125 @@ You can build an in-meeting app for enabling meeting participants to sign docume
 
 You can use an in-meeting signing app to:
 
-- Add documents to be reviewed during a meeting
-- Share documents to be reviewed to main stage
-- Sign documents using the signer’s identity
+* Add documents to be reviewed during a meeting
+* Share documents to be reviewed to main stage
+* Sign documents using the signer’s identity
 
 The participants can review and sign documents, such as purchase agreements and purchase orders.
 
 :::image type="content" source="../assets/images/sbs-inmeeting-doc-signing/final-output.png" alt-text="In-meeting document signing app":::
 
 The following participant roles may be involved during the meeting:
 
-- **Document creator**: This role can add their own documents to be reviewed and signed.
-- **Signer**: This role can sign reviewed documents.
-- **Reader**: This role can view the documents added to the meeting.
+* **Document creator**: This role can add their own documents to be reviewed and signed.
+* **Signer**: This role can sign reviewed documents.
+* **Reader**: This role can view the documents added to the meeting.
+
+## App caching
+
+App caching helps you to improve launch time of the apps that are loaded in the meeting side panel.
```

Comment:
App Caching is a feature available to Teams App Developers to help make subsequent launches of Apps faster in Meetings Side Panel. App Caching support will be extended to other surfaces (i.e. Chats, Channels, Personal Apps) over time.